mergify_engine/queue/merge_train.py

```diff
@@ -1388,7 +1388,8 @@ async def remove_pull(self, ctxt: context.Context) -> None:
         if position is None:
             return
         await self._slice_cars(
-            position, reason="Pull request ahead in queue got removed from the queue"
+            position,
+            reason=f"Pull request ahead in queue #{ctxt.pull['number']} got removed from the queue",
```

Comment:
This looks confusing. Did you mean:
```suggestion
            reason=f"Pull request #{ctxt.pull['number']} which was ahead in the queue has been dequeued",
```
?